### PR TITLE
Mitigate xmldiff package upgrade problem. #10417

### PR DESCRIPTION
**Pull request by ggbecker**
Mitigate xmldiff package upgrade problem.

Version 2.6 of this package triggers #10417 which blocks PRs to get merged. This fix is a temporary solution to unblock PRs.

- Mitigates #10417

### CODE REVIEW

**Comment by ggbecker** [comment]
@Mab879 we need to merge this one ASAP to unblock other PRs to get merged.

**Review by Mab879 — APPROVED** [review]
Once the CI passes I will merge.

**Comment by Mab879** [comment]
Can you please backport this to stabilization as well? We will have PRs that we don't want to block.

**Pull request merged**
